gapis/api/vulkan/api/renderpass_framebuffer.api

```diff
@@ -366,7 +366,7 @@ sub ref!RenderPassObject createRenderPassObjectFromInfo2(
   attachments := info.pAttachments[0:info.attachmentCount]
   for i in (0 .. info.attachmentCount) {
     attachment := attachments[i]
-    renderPass.AttachmentDescriptions[i] = AttachmentDescription(
```

Comment:
I am having hard time to see how changing this solves the problem

Reply:
I think @AWoloszyn gave a pretty thorough answer, but the short version is:

I need to might need to modify this struct when processing the pNext loop so I assign it to a local variable. Then, once I have the final version of the struct, I can put it in the map. Modifying it in-place in the map (the original code) won't work because the map access or will give me a copy.